docs/modules/ROOT/partials/getting-started/proc-configuring-registry-ui.adoc

```diff
@@ -0,0 +1,36 @@
+
+[id="configuring-registry-ui"]
+= Configuring {registry} User Interface
+
+You may configure the {registry} User Interface in a number of ways, either to customize its behavior or to properly
+configure it for your deployment environment.
+
+== Configuring {registry} UI for Deployment Environment
+
+When a user navigates their browser to the {registry} User Interface, some initial configuration settings are loaded.
+Two important configuration properties are:
+
+* URL of the back-end API
+* URL of the front-end user interface
+
+Typically {registry} will automatically detect and generate these settings, but there are some deployment environments
+where this automatic detection may fail.  When this happens, you can configure the following environment variables to
+explicitly set them:
+
+* *_REGISTRY_UI_CONFIG_APIURL_* : set to override the URL to the back-end API (e.g. https://registry.my-domain.com/api)
+* *_REGISTRY_UI_CONFIG_UIURL_* : set to override the URL to the front-end user interface (e.g. https://registry.my-domain.com/ui)
+
+== Configuring {registry} UI for Read-Only Mode
```

Comment:
Suggest sentence case for all headings: 
Configuring the {registry} web console for read-only mode